src/schema/firefox-schemas-import.js

```diff
@@ -324,9 +347,35 @@ function loadSchemasFromFile(basePath) {
   return schemas;
 }
 
-export function importSchemas(firefoxPath, ourPath) {
+export function importSchemas(firefoxPath, ourPath, importedPath) {
   const rawSchemas = loadSchemasFromFile(firefoxPath);
   const ourSchemas = readSchema(ourPath, 'manifest.json');
   const processedSchemas = processSchemas(rawSchemas, ourSchemas);
-  writeSchemasToFile(firefoxPath, processedSchemas);
+  writeSchemasToFile(firefoxPath, importedPath, processedSchemas);
+}
+
+function downloadUrl(version) {
+  return `https://hg.mozilla.org/mozilla-central/archive/FIREFOX_AURORA_${version}_BASE.tar.gz`;
```

Comment:
I only see tags for aurora on [mozilla-central](https://hg.mozilla.org/mozilla-central/tags), but it's possible I'm looking in the wrong spot. I'm guessing those will start being FIREFOX_BETA_<version>_BASE.tar.gz at some point.

We still need to run this script manually for the time being so when the URL changes we can update it and run it.